src/utils/fxa.ts

```diff
@@ -47,34 +46,47 @@ async function destroyOAuthToken(
 
   try {
     const response = await fetch(tokenUrl, tokenOptions);
-    if (!response.ok)
-      throw new InternalServerError(`bad response: ${response.status}`);
-    await response.json();
-    return;
+    const responseJson = await response.json();
+    if (!response.ok) throw new Error(responseJson);
+    console.info("destroy_oauth_token_success");
+    return true;
   } catch (e) {
     if (e instanceof Error) {
```

Comment:
I noticed elsewhere that you re-threw if it wasn't an `Error`, do you think that's worth doing here? If it's not an instance of `Error` then it will just silently return `false` otherwise right?

Reply:
Changed. Though I don't think it's a big deal here. 
It depends on the context, for destroying token, failing silently and logging is probably what we want. We won't be throwing the error back to the user and ask users to do a different action